### ISSUE TITLE
Leaving the Camera Types array blank results in a null ref exception

### ISSUE THREAD

**Issue by djdd87** [djdd87/SynoAI#131]
It looks as though not specifying any types causes an exception:

> I just started using the SynoAI setup which works great for a few image processes, but then it crashes. I have noticed the following error message in the log of SynoAI right before it crashes:
> 
> 2022-06-06 07:58:33 | stdout | Storing keys in a directory '/root/.aspnet/DataProtection-Keys' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.
> -- | -- | --
> 2022-06-06 07:58:33 | stdout | [?1h=[40m[1m[33mwarn[39m[22m[49m: Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository[60]
> 2022-06-06 07:58:27 | stdout | at System.Threading.ThreadPoolWorkQueue.Dispatch()
> 2022-06-06 07:58:27 | stdout | at System.Threading.QueueUserWorkItemCallback.Execute()
> 2022-06-06 07:58:27 | stdout | at System.Threading.ExecutionContext.RunForThreadPoolUnsafe[TState](ExecutionContext executionContext, Action`1 callback, TState& state)
> 2022-06-06 07:58:27 | stdout | at System.Threading.QueueUserWorkItemCallback.<>c.<.cctor>b__6_0(QueueUserWorkItemCallback quwi)
> 2022-06-06 07:58:27 | stdout | at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_1(Object state)
> 2022-06-06 07:58:27 | stdout | at SynoAI.Controllers.CameraController.Get(String id) in /src/Controllers/CameraController.cs:line 211
> 2022-06-06 07:58:27 | stdout | at System.String.Join(String separator, IEnumerable`1 values)
> 2022-06-06 07:58:27 | stdout | Unhandled exception. System.ArgumentNullException: Value cannot be null. (Parameter 'values')
> 2022-06-06 07:58:27 | stdout | FrontDoor: Nothing detected by the AI at EVENT TIME 972ms
> 
> Any ideas?

_Originally posted by @TullingeHellMoppers in https://github.com/djdd87/SynoAI/discussions/130_

**Comment by TullingeHellMoppers** [comment]
Hello

Have you had time to look at my other issue for the Webhook? Would be great
if I could get that working so that I can trigger the recording in the
Surveillance station that way.

Thanks,
David

**Comment by djdd87** [comment]
Config from discussion:

```json
"Cameras": [
{
  "Name": "FrontDoor",
  "Types": [ ],
  "Threshold": 30,
  "MinSizeX": 50,
  "MinSizeY": 50
}
```

**Comment by TullingeHellMoppers** [comment]
I figured out what was wrong.NOT WORKING:{      "Type": "Webhook",      "Url": "http://192.168.0.166:5000/webapi/entry.cgi?api=SYNO.SurveillanceStation.Webhook&method=Incoming&version=1",      "Method": "GET",      "Authentication": "Bearer",      "Token": "Xxxxxx",      "SendImage": "false",	  "Cameras": [ "Cabin"]    },	WORKING:	{      "Type": "Webhook",      "Url": "http://192.168.0.166:5000/webapi/entry.cgi?api=SYNO.SurveillanceStation.Webhook&method=Incoming&version=1&token=xxxxxxxx",      "Method": "GET",      "SendImage": "false",	  "Cameras": [ "Cabin"]    },No Bearer Auth should be configured.Not sure if the ReadMe is incorrect or applies to certain conditions.But still, would be nice to see the Webhook response body in the Log, atleast when in Debug.Thanks allot for this code, now my setup detects persons and then triggers a recording. Should result in significantly less false triggers.
-------- Original message --------From: djdd87 ***@***.***> Date: 6/14/22  13:24  (GMT+01:00) To: djdd87/SynoAI ***@***.***> Cc: TullingeHellMoppers ***@***.***>, Mention ***@***.***> Subject: Re: [djdd87/SynoAI] Leaving the Camera Types array blank results in a null ref exception (Issue #131) 
Config from discussion:
"Cameras": [
{
  "Name": "FrontDoor",
  "Types": [ ],
  "Threshold": 30,
  "MinSizeX": 50,
  "MinSizeY": 50
}

**Comment by djdd87** [comment]
There's nothing wrong with the README, it's an *example*. You need to configure your call according to what your webhook requires.

**Comment by djdd87** [comment]
Fixed in `v1.2.0`

**Comment by TullingeHellMoppers** [comment]
Thanks! But I need to see the response Body message. See here how Synology is returning False in Body but still Http ***@***.***:/$ HTTP/1.1 200 OKServer: nginxDate: Tue, 14 Jun 2022 13:16:59 GMTContent-Type: application/json; charset="UTF-8"Transfer-Encoding: chunkedConnection: keep-aliveKeep-Alive: timeout=20X-Content-Type-Options: nosniffX-XSS-Protection: 1; mode=blockCache-Control: max-age=0, no-cache, no-store, must-revalidatePragma: no-cacheExpires: 0{"error":{"code":101},"success":false}Still do not understand what is failing....
-------- Original message --------From: djdd87 ***@***.***> Date: 6/14/22  13:52  (GMT+01:00) To: djdd87/SynoAI ***@***.***> Cc: TullingeHellMoppers ***@***.***>, Mention ***@***.***> Subject: Re: [djdd87/SynoAI] Leaving the Camera Types array blank results in a null ref exception (Issue #131) 
Closed #131 as completed.

**Comment by djdd87** [comment]
Sorry, don't understand what you mean. This issue was to fix a null reference exception, which is now fixed. If you have another issue, please raise it separately.
